src/App.tsx

```diff
@@ -10,33 +11,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const onTabSelected = (tabData: TabsType) => {
```

Comment:
```suggestion
  const handleTabSelected = (tabData: TabsType) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364